# ==============================================================================
# FIXING COMMIT DIFF
# ==============================================================================
# File: js/src/jit-test/tests/parser/bug-1764737.js
# Commit: b4f304b80390
# Full Hash: b4f304b803904a9d73cd9c69b0ed66f78db5c039
# Author: Jonatan Klemets <jonatan.r.klemets@gmail.com>
# Date: 2022-04-14 21:45:12
# Description:
#   Bug 1764737 - Add missing OOM checks in GeneralParser<ParseHandler, Unit>::exportFrom. r=mgaudet
#   
#   Differential Revision: https://phabricator.services.mozilla.com/D143751
# ==============================================================================

diff -r ed4101ff3fae -r b4f304b80390 js/src/jit-test/tests/parser/bug-1764737.js
--- /dev/null	Thu Jan 01 00:00:00 1970 +0000
+++ b/js/src/jit-test/tests/parser/bug-1764737.js	Thu Apr 14 19:08:03 2022 +0000
@@ -0,0 +1,9 @@
+// |jit-test| skip-if: !('oomTest' in this); --fuzzing-safe; --ion-offthread-compile=off
+
+function r(src) {
+  oomTest(function() {
+      parseModule(src);
+  });
+}
+r("export * from 'y';");
+r("export * from 'y';");
